lib.deno.ns.d.ts

```diff
@@ -185,8 +185,7 @@ declare namespace Deno {
    */
   export function osRelease(): string;
 
-  /** Exit the Deno process with optional exit code. If no exit code is supplied
-   * then Deno will exit with return code of 0.
+  /** 退出Deno进程，可以指定退出码，若无则为 0.
```

Comment:
```suggestion
  /** 退出 Deno 进程，可以指定退出码，若无则为 0。
```

中文和英文之间加空格，末尾句号使用中文句号。